### PR TITLE
Bluetooth: Controller: Fix lll ISO stream get by group

### PR DESCRIPTION
**Pull request by cvinayak**
Fix LLL ISO stream get by group to return stream context sorted by CIS offset in ascending order.

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/ull_conn_iso.c [hunk 1]
```diff
@@ -300,6 +300,57 @@ ull_conn_iso_lll_stream_get_by_group(struct lll_conn_iso_group *cig_lll,
 	return &cis->lll;
 }
 
+struct lll_conn_iso_stream *
```
Comment (mtpr-ot) [inline subsys/bluetooth/controller/ll_sw/ull_conn_iso.c:303]: Maybe a comment about how it's sorting? It's not obvious from the name

Comment (cvinayak) [reply subsys/bluetooth/controller/ll_sw/ull_conn_iso.c:313]: Added comments.

**Review by mtpr-ot — APPROVED** [review]
Looks good

**Comment by cvinayak** [comment]
@henrikbrixandersen @MaureenHelm If this PR gets approvals before the final tag cut, it will improve the Bluetooth Controller' LE audio interoperability with other implementations.

**Review by jori-nordic — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**